### PR TITLE
Move Fabric from react-next to react-internal

### PR DESCRIPTION
**Pull request by tomi-msft**
#### Pull request checklist

- [x] Addresses an existing issue: Part of #15152 
- [x] Include a change request file using `$ yarn change`

This PR moves the version of `Fabric` existing in `@fluentui/react-next` to `@fluentui/react-internal`. As part of this API and snapshots were updated, and the versions of these components in @fluentui/react-next now re-export the versions in @fluentui/react-internal.

#### Focus areas to test

(optional)

### CODE REVIEW

File: packages/react-internal/src/components/Fabric/Fabric.test.tsx [hunk 1]
```diff
@@ -1,84 +1,78 @@
...
-import * as ReactDOM from 'react-dom';
-import * as renderer from 'react-test-renderer';
-import { mount } from 'enzyme';
+import { create } from '@uifabric/utilities/lib/test';
+import { Customizer } from '@uifabric/utilities';
 import { Fabric } from './Fabric';
-// import { Checkbox } from '../../Checkbox';
-// import { Customizer } from '../../Utilities';
-// import { createTheme } from '../../Styling';
+import { Checkbox } from '@fluentui/react/lib/Checkbox';
```
Comment (tomi-msft) [inline packages/react-internal/src/components/Fabric/Fabric.test.tsx:5]: @dzearing This component is not present in react-internal. What component within react-internal would you recommend using instead?

Comment (ecraig12345) [reply packages/react-internal/src/components/Fabric/Fabric.test.tsx:5]: For now let's just comment out this import and the parts of the test that use it, unless someone has a suggestion right away for another component to use.

**Comment by ecraig12345** [comment]
You'll probably need to update snapshots in `react-internal` to get the build to pass.

**Review by ecraig12345 — APPROVED** [review]

**Pull request merged**